@navikt/core/css/darkside/accordion.darkside.css

```diff
@@ -1,198 +1,168 @@
+      background-color: var(--ax-bg-accent-strong-hover);
+      color: var(--ax-text-accent-contrast);
+    }
+
+    @media (forced-colors: active) {
+      background-color: canvas;
+      border: 1px solid highlight;
+      color: highlight;
+    }
+  }
 
-.navds-accordion__item:last-child > :where(.navds-accordion__header) {
-  box-shadow:
-    var(--__ac-accordion-header-shadow),
-    inset 0 -2px 0 0 var(--__ac-accordion-header-shadow-color);
-}
+  &:focus-visible {
+    outline: 2px solid var(--ax-border-focus);
+    outline-offset: 2px;
+    z-index: 1;
 
-.navds-accordion__item:last-child:where(.navds-accordion__item--open) :where(.navds-accordion__header) {
-  box-shadow:
-    inset 2px 0 0 0 var(--a-transparent),
-    inset -2px 0 0 0 var(--a-transparent),
-    inset 0 2px 0 0 var(--a-transparent),
-    inset 0 2px 0 0 var(--__ac-accordion-header-shadow-color);
+    &::before,
+    &::after {
+      display: none;
+    }
+  }
 }
 
-.navds-accordion__item:last-child:where(.navds-accordion__item--open) {
-  box-shadow:
-    inset 2px 0 0 0 var(--a-transparent),
-    inset -2px 0 0 0 var(--a-transparent),
-    inset 0 2px 0 0 var(--a-transparent),
-    inset 0 -2px 0 0 var(--__ac-accordion-header-shadow-color);
+.navds-accordion__header-content {
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
-.navds-accordion__item--open > :where(.navds-accordion__header) {
-  box-shadow: var(--__ac-accordion-header-shadow);
+.navds-accordion__header::before,
+.navds-accordion__header::after,
+.navds-accordion__item:last-child > .navds-accordion__content::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 1px;
+  background-color: var(--ax-border-subtle);
 }
 
-.navds-accordion__header:focus-visible {
-  outline: 2px solid transparent;
-  outline-offset: 3px;
-  box-shadow: var(--a-shadow-focus);
-  border-radius: var(--a-border-radius-large);
+.navds-accordion__header::after,
+.navds-accordion__item:last-child > .navds-accordion__content::after {
+  top: initial;
+  bottom: 0;
 }
 
-@supports not selector(:focus-visible) {
-  .navds-accordion__header:focus {
-    outline: none;
-    box-shadow: var(--a-shadow-focus);
-    border-radius: var(--a-border-radius-large);
-  }
+.navds-accordion__item[data-expanded="true"] > .navds-accordion__header::after,
+.navds-accordion__item:not(:last-child) > .navds-accordion__header::after {
+  display: none;
 }
 
-.navds-accordion__header-content {
-  overflow: hidden;
-  text-overflow: ellipsis;
+.navds-accordion__item[data-expanded="false"]:has(.navds-accordion__header:where(:hover, :focus-visible))
+  + .navds-accordion__item
+  > .navds-accordion__header::before {
+  display: none;
 }
 
-/*************************
- * Icon *
- *************************/
+/* ------------------------- Accordion chevron icon ------------------------- */
 .navds-accordion__icon-wrapper {
   display: grid;
   place-content: center;
```

Comment:
The wrapper itself or border-radius? As for the former, probably not, but trying to avoid introducing breaking changes into the react-code for Accordion

Reply:
I'm referring to this:
```css
  display: grid;
  place-content: center;
```

I selected those lines when commenting, so the comment should say "Comment on lines 98 to 99", but I see that it can be unclear when viewing the comment in the "Files changed" tab.